runbot/models/build_error.py

```diff
@@ -511,6 +515,18 @@ def action_clean_content(self):
         for errors_content_to_merge in to_merge:
             errors_content_to_merge._merge()
 
+    def _find_duplicates(self):
+        fingerprints = [r['fingerprint'] for r in self.env['runbot.build.error.content'].read_group([], ['id_count:count(id)', 'fingerprint'], ['fingerprint']) if r['id_count'] >1]
+        duplicates = self.env['runbot.build.error.content'].search([('fingerprint', 'in', fingerprints)], order='fingerprint')
+        return {
+            "type": "ir.actions.act_window",
+            "res_model": "runbot.build.error.content",
+            "domain": [('id', 'in', duplicates.ids)],
+            "context": {"create": False, 'group_by': ['fingerprint']},
+            "name": "Duplicate Error contents",
+            'view_mode': 'tree'
+        }
```

Comment:
This achieves the same with a singular query to the database and less data transfer
```suggestion
        rg = self.env['runbot.build.error.content']._read_group(
            [],
            ['fingerprint'],
            ['id_count:count(id)', 'id_arr:array_agg(id)'],
            [('id_count', '>', 1)]
        )
        return {
            "type": "ir.actions.act_window",
            "res_model": "runbot.build.error.content",
            "domain": [('id', 'in', [content_id for r in rg for content_id in r[2]])],
            "context": {"create": False, 'group_by': ['fingerprint']},
            "name": "Duplicate Error contents",
            'view_mode': 'tree'
        }
```